share/xslt/checks/check-git.xsl

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are you sure you want to commit that? (Given how we changed the schema, there is no necessity for this check anymore.)

Reply:
I know, it was kind of a "dummy stylesheet" which acts like a blueprint for other checks. :wink:

It wasn't finished yet, but I wanted to create a snapshot of it before my vacation.